src/routing_node.rs

```diff
@@ -348,6 +352,11 @@ impl RoutingNode {
             return Err(RoutingError::BadAuthority);
         }
 
+        let (message, opt_token) = match self.accumulate(message_wrap) {
```

Comment:
Sorry no :), this function does not exclude this one message type; can we patch this here?